### PR TITLE
Added More Event Block Support

### PR DESCRIPTION
**Pull request by WilsontheWolf**

### CODE REVIEW

**Comment by WilsontheWolf** [comment]
Fixed and ready to merge @AlexejheroYTB

File: www/css/dark.css [hunk 1]
```diff
@@ -31,6 +32,10 @@ body {
   color: var(--disabled-color) !important;
 }
 
+a.gray {
+  color: var(--disabled-color);
+}
```
Comment (Alexejhero) [inline www/css/dark.css]: Why did you add this? This was not needed. There is another rule (`.gray`) that does this.

Comment (WilsontheWolf) [reply www/css/dark.css]: merging commits. I had it you didn't so I just put it in
